lib/opte/src/engine/flow_table.rs

```diff
@@ -229,16 +224,15 @@ fn flow_expired_probe(
     last_hit: Option<Moment>,
     now: Option<Moment>,
 ) {
+    let a = crate::NopDrop;
```

Comment:
All `NopDrop` code will be removed once illumos#16480 is merged in and available on the `helios-2.0` lab image.